### ISSUE TITLE
API Platform: Unable to generate an IRI for "App\Entity\Taxonomy\Taxon".

### ISSUE THREAD

**Issue by tsogoo** [Sylius/Sylius-Standard#494]
I'm using Sylius-1.9.0-alpha2. And new /api/v2/shop/taxons and /api/v2/shop/products routes response body shows:
```
{
  "code": 500,
  "message": "Unable to generate an IRI for \"App\\Entity\\Taxonomy\\Taxon\"."
}
```

**Comment by tsogoo** [comment]
To fix downgrade api-platform to 2.5.9 or upgade sylius1.9-alpha to sylius1.9-beta.

**Comment by yvipin6887** [comment]
Hi There,
 I am facing same issue with Sylius-1.12 version.
